config/manager/controller-config.yaml

```diff
@@ -4,9 +4,7 @@ metadata:
   name: numaplane-controller-config
 data:
   config.yaml: |
-    syncTimeIntervalMs: 30000
+    logLevel: 1
```

Comment:
@afugazzotto I believe level 3 would be INFO, wouldn't it?

Reply:
is 1 debug?

Reply:
no, 4 is debug

Reply:
We had an issue in the previous repo to be able to use names instead of numbers. I am going to add that here too.

Reply:
yeah, we should default to info then.

Reply:
@xdevxy i guess this PR is good to merge .Please have a look